.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+// For more details, see https://aka.ms/devcontainer.json.
+{
+    "build": {
+        "context": ".",
+        "dockerfile": "Dockerfile"
+    },
+
+    // Uncomment to install NVIDIA CUDA - required for a GPU-powered codespace
+    // "features": {
+    //     "ghcr.io/devcontainers/features/nvidia-cuda:1": {
+    //         "installCudnn": true
+    //     }
+    // },
+
+    "hostRequirements": {
```

Comment:
There's a way to require a GPU sku, right? Should we have that commented out here too for folks that would want to level up to a GPU codespace?

Reply:
I have already added the details to the getting-started-with-codespaces.md doc.
Linked in here as well. thanks!